csharp/ql/lib/upgrades/ba2201248071b2bf0bb52909b35014091d2e18a6/attribute_kind.ql

```diff
@@ -0,0 +1,16 @@
+class Attribute extends @attribute {
+  string toString() { none() }
+}
+
+class Attributable extends @attributable {
+  string toString() { none() }
+}
+
+class TypeOrRef extends @type_or_ref {
+  string toString() { none() }
+}
+
+query predicate add_default_kind(Attribute id, int kind, TypeOrRef type_id, Attributable target) {
```

Comment:
hah, I was wondering the same thing, but thought it was safe, since the CI didn't complain.
Maybe the predicate has to be named *select* or something similar :-)
In any case, I will change it according to your recommendation!